File: solutions/2248-minimum-cost-of-buying-candies-with-discount/minimum-cost-of-buying-candies-with-discount.go
func minimumCost(cost []int) int {
	sort.Sort(sort.Reverse(sort.IntSlice(cost)))
	res := 0
	n := len(cost)
	for i := 0; i < n; i++ {
		if i%3 != 2 {
			res += cost[i]
		}
	}
	return res
}

File: solutions/2248-minimum-cost-of-buying-candies-with-discount/minimum-cost-of-buying-candies-with-discount.js
var minimumCost = function (cost) {
    cost.sort((a, b) => b - a);
    let res = 0;
    const n = cost.length;
    for (let i = 0; i < n; ++i) {
        if (i % 3 !== 2) {
            res += cost[i];
        }
    }
    return res;
};